src/apicast-cli/blank-app/.circleci/config.yml.liquid

```diff
@@ -0,0 +1,15 @@
+version: 1
```

Comment:
it should be `version: 2` as https://github.com/3scale/apicast/commit/9ef1b48e41581844b13711dd47fab63bc7a78775